OpenUpSA/wazimap-ng-ui
# added demarcation warning

## Description
adding a warning to let user know that only the default version data is displayed in the bottom right of tabular comparison

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-792

## How is it tested automatically?


## How should a reviewer test it locally
* go to tabular comparison
* confirm that the warning is displayed
* confirm that it shows the correct version name


## Screenshots
<img width="1494" alt="Screenshot 2023-09-20 at 10 21 21" src="https://github.com/OpenUpSA/wazimap-ng-ui/assets/53019884/0dbb1d69-b033-4fd3-a98a-f585ca923399">


## Changelog

### Added

### Updated
* `src/js/tabular_comparison/body.js` to get the default version name
* `src/js/tabular_comparison/result.js` to display the warning

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
